### PR TITLE
writing the code of services section

### LINKED ISSUE #1: Services-section-in-home-page
### Main Task

-Writing html and css code for 3 divs ; one for "buy", second for "rent" and the last for see "neighborhoods".
-

### PR DESCRIPTION
**Pull request by ahmadfarid-059**
Rlates #1
writing the code of services section

### CODE REVIEW

**Review by nurabunamus — DISMISSED** [review]

File: style.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+.descriptive_paragraph{
+    color: #071144;
+    font-size: 15px;
+    
+    width: 100%;
+
+    text-align: center;
+    
+    
+}
```
Comment (AlaaTaima) [inline style.css]: **update the width here to `80%`**
```suggestion
.descriptive_paragraph{
    color: #071144;
    font-size: 15px;
    
    width: 80%;

    text-align: center;
    
    
}
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+.divs{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    display: inline-block;
+    text-align: center;
+}
```
Comment (AlaaTaima) [inline style.css]: **update the style here as the following to enhance the final results :** 
```suggestion
.divs{
    display: flex;
    flex-direction : column;
    justify-content: space-around;
    align-items: center;
    height: 85vh;
    padding: 30px 0px;
}
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    .main_div{
+        display: flex;
+        flex-direction: column;
+        height: auto;
+
+    }
+    .descriptive_paragraph{
+        width: 90%;
+        text-align: center;
+    }
```
Comment (AlaaTaima) [inline style.css]: **Ther is no need for this style here , delete it.**

**For these reasons :** 

* Delete `width` : the default width is suitable.
* It has already taken ` text-align: center;` So, delete it too.
---
**Replace it with :** 

```
    .title{
        font-size: 18px;

}
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    text-align: center;
+}
+/* media quiery */
+@media screen and (max-width: 768px) {
+    .main_div{
+        display: flex;
+        flex-direction: column;
+        height: auto;
+
+    }
```
Comment (AlaaTaima) [inline style.css]: **`display:flex` has already taken so, just you have to add the property `flex-direction`** 
```suggestion
    .main_div{
        flex-direction: column;
        height: auto;

    }
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    width: 180px;
+    height: 50px;
+    border-radius: 15%;
+    background-color: #0f4b5e;
+    color: white;
+    font-size: 20px;
+    font-weight: lighter;
+    margin-top: 10px;
+
+}
```
Comment (AlaaTaima) [inline style.css]: **Remove the default border of the button by using `border : none`** 

```suggestion
.button{
    width: 180px;
    height: 50px;
    border-radius: 15%;
    background-color: #0f4b5e;
    color: white;
    font-size: 20px;
    font-weight: lighter;
    margin-top: 10px;
    border : none;

}
```

File: style.css [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+.button:hover{
+    background-color: white;
+    color: cornflowerblue;
+}
```
Comment (AlaaTaima) [inline style.css]: **change the mouse shape to pointer by using `cursor: pointer;`**
```suggestion
.button:hover{
    background-color: white;
    color: cornflowerblue;
        cursor : pointer;

}
```

File: style.css [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+    .title{
+        font-size: 20px;
+
+}
+@media screen and (max-width: 340px) {
+    .title{
+        font-size: 18px;
+        font-weight: bold;
+}
+}
```
Comment (AlaaTaima) [inline style.css]: **No need for this query , delete it**

File: style.css [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+    
+}
+.title{
+    margin: 20px;
+    color: #071144;
+    padding-top: 15px;
+    font-weight: bold;
+
+    
+}
```
Comment (AlaaTaima) [inline style.css]: **Add a `width : 100%` to the title style here :** 
```suggestion
.title{
    margin: 20px;
    color: #071144;
    padding-top: 15px;
    font-weight: bold;
    width : 100%;

    
}
```

File: style.css [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+    margin-top: 10px;
+
+}
+.button:hover{
+    background-color: white;
+    color: cornflowerblue;
+}
+a:link{
+    text-decoration: none;
+}
```
Comment (AlaaTaima) [inline style.css]: **Fix the selector here :** 
```suggestion
a {
    text-decoration: none;
}
```

**Review by AlaaTaima — CHANGES_REQUESTED** [review]
**Good Job 👍** 

**But, there are some points that would be awesome to get enhanced.**

**Review by AlaaTaima — APPROVED** [review]

**Pull request merged**